### PR TITLE
Premium 1.9

### PR DESCRIPTION
**Pull request by dzucconi**
coupons-cta-2

Looks like the about page pricing table is pretty much the only thing that's not deployable, interestingly. I'll merge this on Monday after double checking everything.

### CODE REVIEW

**Comment by dzucconi** [comment]
- [ ] Blog post links
- [ ] Pop-up messages

**Comment by dzucconi** [comment]
✔️

**Comment by dzucconi** [comment]
@broskoski should be good to go on staging. Whatchu think?

File: apps/settings/components/coupon_code/view.coffee [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  events:
+    'input input': 'onInput'
+
+  initialize: ->
+    @listenTo @model, 'change:description', @render
+    @listenTo @model, 'change:code', @render
+    @listenTo @model, 'change:code', debounce(@checkCode, 500)
+
+  onInput: ->
+    code = @els.input.val()
```
Comment (broskoski) [inline apps/settings/components/coupon_code/view.coffee]: Where is `@els` coming from here?

Comment (dzucconi) [inline apps/settings/components/coupon_code/view.coffee]: Look above. Just an object. Just my own stylistic quirks. I prefer this over `@$input`.

Comment (broskoski) [inline apps/settings/components/coupon_code/view.coffee]: Right I see the empty object but can't find where it gets set?

Comment (broskoski) [inline apps/settings/components/coupon_code/view.coffee]: Nevermind, I get it.

File: components/blocks/container/templates/_grid_contents.jade [hunk 2]
```diff
@@ -1,5 +1,13 @@
 if can && can.add_to
   include ../../grid_item/templates/types/new_block
 
+else if channel && channel.get('status') === 'private'
+  //- If we can't add to the channel but it's private then that means
+  //- the user needs to upgrade to premium to add to it
+  //-
+  //- The reason this only exists in grid view is because
+  //- list view is premium-only so this will never be seen in that view mode.
+  include ../../../premium_cta_block/index
```
Comment (broskoski) [inline components/blocks/container/templates/_grid_contents.jade]: This should not be here yet, right?

Comment (dzucconi) [inline components/blocks/container/templates/_grid_contents.jade]: No it's fine; it's not possible to see until we change the ability model. `status === 'private' && !can.add_to` can't happen until then.

Comment (broskoski) [inline components/blocks/container/templates/_grid_contents.jade]: Ah, got it.

**Pull request merged**